taskt/Core/Settings/WhiteListIPSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace taskt.Core.Settings
+{
+    [Serializable]
+    public class WhiteListIPSettings
+    {
+        public string Value { get; set; }
+        public WhiteListIPSettings(string s)
```

Comment:
Can you please change this parameter name `string s` to a meaningful variable name.